lib/LWP/Protocol/Net/Curl.pm

```diff
@@ -217,6 +217,18 @@ sub _handle_method {
         },
     );
 
+	if (my $protocol = $request->protocol) {
+		my $v;
+		if ($protocol =~ m/^HTTP\/1\.0$/) {
+			$v = CURL_HTTP_VERSION_1_0;
+		} elsif ($protocol =~ m/^HTTP\/1\.1$/) {
+			$v = CURL_HTTP_VERSION_1_1;
+		} elsif ($protocol =~ m/^HTTP\/2$/) {
+			$v = CURL_HTTP_VERSION_2TLS;
```

Comment:
Please note that the `CURL_HTTP_VERSION_2TLS` symbol does not exist in libcurl prior to 7.47.0. It is worth checking within an `eval` if it is available at all. On the other hand, such check does not need to happen on every `_handle_method` call; it should be done during the module load.
Sorry for being so picky!